src/inventory_ui.cpp

```diff
@@ -2763,6 +2811,15 @@ inventory_input inventory_selector::process_input( const std::string &action, in
                 if( res.entry != nullptr && res.entry->is_selectable() ) {
                     return res;
                 }
+                if( res.entry == nullptr && res.action == "SELECT" ) {
+                    p.x++;
+                    res.entry = find_entry_by_coordinate( p );
+                    // if the user has clicked the coordinate just to the left of a pocket, they are clicking the pocket's expander icon
+                    if( res.entry != nullptr && res.entry->is_selectable() && res.entry->chevron ) {
+                        res.action = "SHOW_HIDE_CONTENTS";
+                        return res;
+                    }
+                }
```

Comment:
This won't work for `inventory_multiselector` https://github.com/CleverRaven/Cataclysm-DDA/blob/6ef8d469bf1d2e0f8e729bd372973d86fad53342/src/inventory_ui.cpp#L3553-L3556

Either add another special case there or move this code to `inventory_column::on_input()`

Reply:
I like the idea of moving the code into inventory_column::on_input, however find_entry_by_coordinate is a member of inventory_selector. We would need to add more members to inventory_input so that subclasses could handle this. If you find this approach preferable, I will go this route.

Reply:
AFAICT you're solving this one in the hover PR and it only needs a tiny change there.

Reply:
after merging in master with the mouse-hover branch changes, this appears to be fixed, resolving.